doc/api/cli.md

```diff
@@ -816,6 +816,33 @@ added: v12.0.0
 Set default [`tls.DEFAULT_MIN_VERSION`][] to 'TLSv1.3'. Use to disable support
 for TLSv1.2, which is not as secure as TLSv1.3.
 
+### `--trace-atomics-wait`
+<!-- YAML
+added: REPLACEME
+-->
+
+Print short summaries of calls to [`Atomics.wait()`][] to stderr.
+The output could look like this:
+
+```text
+(node:15701) [Thread 0] Atomics.wait(<address> + 0, 1, inf) started
```

Comment:
We might as well just don't print the timeout if it's infinity to match what you'd expect the call would look like

Reply:
I don’t mind being explicit here.

Reply:
I prefer being explicit here also. The only think I would worry about is using the abbreviation `inf` for `Infinity`. It's a minor issue, however, as folks who would typically be looking at this should be sophisticated enough in their understanding of the mechanism to know what `inf` means.